ingestion_tools/dataset_configs/template.yaml

```diff
@@ -93,6 +93,16 @@ annotations: OPTIONAL
       delimiter: OPTIONAL, STRING (DEFAULT ',')
     parent_filters: see InstanceSegmentation.parent_filters
     exclude: SEE InstanceSegmentation.exclude
+  - InstanceSegmentationMask:
+      file_format: see InstanceSegmentation.file_format
+      glob_string: see InstanceSegmentation.glob_string
+      glob_strings: see InstanceSegmentation.glob_strings
+      is_visualization_default: see InstanceSegmentation.is_visualization_default
+      is_portal_standard: OPTIONAL, BOOLEAN (DEFAULT FALSE)
+      scale_factor: OPTIONAL, FLOAT (DEFAULT 1) (POSITIVE)
+      mask_label: OPTIONAL, INTEGER (DEFAULT 1)
```

Comment:
As a reminder, this one doesn't really make sense because there would be multiple labels.